front/lib/api/assistant/configuration.ts

```diff
@@ -200,37 +204,81 @@ export async function createAgentConfiguration(
     status,
     generation,
     action,
+    sId,
   }: {
     name: string;
     description: string;
     pictureUrl: string;
     status: AgentConfigurationStatus;
     generation: AgentGenerationConfigurationType | null;
     action: AgentActionConfigurationType | null;
+    version?: number;
```

Comment:
version is added here but not above should be removed?

Reply:
Indeed, a leftover from prev version